src/libraries/Directory.Build.props

```diff
@@ -4,6 +4,10 @@
   </PropertyGroup>
   <Import Project="..\..\Directory.Build.props" />
 
+  <PropertyGroup>
+    <AssemblyVersion>5.0.0.0</AssemblyVersion>
```

Comment:
This matches https://github.com/dotnet/corefx/pull/41723/files#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR5, we should maybe use `ProductVersion` or `Major/Minor/PatchVersion` though.

Reply:
cc: @ericstj I think it makes sense to use any of the recommendations above.

Reply:
Yes, we should not have the version specified in multiple places. 
I'd use `<AssemblyVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)</AssemblyVersion>` if you want to suppress the default (42.42.42.42).

Reply:
I'd be also nice to add a comment that explains why is this set here - something like (guessing) "The AssemblyVersion must match `netcoreapp5.0` moniker. Whenever the version is updated we need to update the TFM as well.".